install.sh

```diff
@@ -455,10 +455,17 @@ setup_selinux() {
         rpm_site="rpm-testing.rancher.io"
     fi
 
-    policy_hint="please install:
+    if awk -F'=' '/ID_LIKE/ {print $2}' /etc/os-release|grep -q "suse"; then
+        policy_hint="k3s with SELinux is currently not supported on SUSE/openSUSE systems.
```

Comment:
This is a static string and should preferably be quoted with single quotes since nothing is to be expanded.